src/main/mocha.ts

```diff
@@ -254,7 +261,21 @@ export default class MochaWrapper extends Mocha {
           file,
           syncedSubprocessData,
         });
-      });
+      }
+
+      test.on('message', onMessageHandler);
+      test.stdout.on('data', onStdoutData);
+      test.stderr.on('data', onStderrData);
+      test.on('close', onClose);
+
+      function clean() {
+        test.removeListener('message', onMessageHandler);
+        test.stdout.removeListener('data', onStdoutData);
+        test.stderr.removeListener('data', onStderrData);
+        test.removeListener('close', onClose);
+        test.destroy();
+        return null;
```

Comment:
Is this `return null` thing a required part of this function? It doesn't look like you're waiting for return value.